mod.sp

```diff
@@ -7,6 +7,53 @@ locals {
   }
 }
 
+variable "common_dimensions" {
+  type        = list(string)
+  description = "A list of common dimensions to add to each control."
+  # Define which common dimensions should be added to each control.
+  # - account_id
+  # - connection_name (_ctx ->> 'connection_name')
+  # - region
+  default = ["account_id", "connection_name", "region"]
+}
+
+variable "tag_dimensions" {
+  type        = list(string)
+  description = "A list of tags to add as dimensions to each control."
```

Comment:
```suggestion
  description = "A list of tags to add as dimensions to each control."
   # A list of tag names to include as dimensions for resources that support
  # tags (e.g. "Owner", "Environment"). Default to empty since tag names are
  # a personal choice.
```